pkg/client/goboapi.go

```diff
@@ -43,3 +48,98 @@ func (api *APIClient) newRequest(graphqlRequest string) *graphql.Request {
 	req.Header.Set("Cache-Control", "no-cache")
 	return req
 }
+
+func extractGraphqlErrorMsgs(errorsInput error) error {
+	if errorsInput != nil {
+		if graphqlErrors, ok := errorsInput.(graphql.Errors); ok {
+			logrus.Debugf("extractGraphqlErrorMsg got %+v", graphqlErrors)
+			errorMsgs := make([]string, len(graphqlErrors))
+			for i, e := range graphqlErrors {
+				errorMsgs[i] = getPrioritizedErrMsg(e)
+			}
+			return errors.New(strings.Join(errorMsgs, "; "))
+		} else {
+			logrus.Warnf("extractGraphqlErrorMsg got ordinary error (deprececated): %s", errorsInput)
+			return errorsInput
+		}
+	} else {
+		return errors.New("extractGraphqlErrorMsg got no error")
+	}
+}
+
+// Extract message from error by prioritised level
+// 1. error.extensions.errors[...].details[...].message
+// 2. error.extensions.errorMessage
+// 3. error.message (default)
+func getPrioritizedErrMsg(graphqlErr graphql.Error) string {
+	extensions, parseError := parseExtensions(graphqlErr.Extensions)
+	if parseError == nil && extensions != nil {
```

Comment:
Blir det enklere å bruke "type-metoder" på extensions og extError for å hente ut denne informashjonen? Jfr det du gjorde på graphq: https://github.com/Skatteetaten/graphql/blob/b4ccc063e40d7df0ba0968794ca0d8181ae331d4/graphql_errors.go#L23